src/audio.rs

```diff
@@ -44,15 +76,20 @@ impl Audio {
         }).unwrap()
     }
 
-    /// Start playing a new sound.
-    pub fn play(&mut self, audio: &asset::Audio) {
-        self.sources.push(AudioSource {
-            position: 0,
-            looping: false,
-            samples: audio.samples(),
+    /// Start playing a new sound once.
+    ///
+    /// This uses the "default" settings for the [AudioSource]. If you want to modify it, use
+    /// [Audio::play_source].
+    pub fn play_asset(&mut self, audio: &asset::Audio) {
```

Comment:
Not sure i like this.

When would you use it? Why not use the default audio play?